### PR TITLE
feat(stats): render MsgLeaseStartReclaim in transaction explorer

### PR DESCRIPTION
**Pull request by baktun14**
## Why

`stats.akash.network` is the public record of on-chain Console activity. After the v2.1.0 network upgrade (AEP-82 resource reclamation), providers submit `MsgLeaseStartReclaim` to begin reclaiming a lease. Without a dedicated component the stats-web transaction explorer renders it through the raw-JSON fallback. This adds a proper component so it renders like every other recognized Akash message type.

Closes CON-379

> [!NOTE]
> The reclamation **deadline** is intentionally out of scope: it lives only on the `EventLeaseReclaimStarted` block event (not on the message), the tx explorer renders message data only, and the indexer work to surface it (CON-287) was cancelled. This PR ships the **lease + provider reason**. CON-379's acceptance criteria should be updated to drop the deadline bullet.

## What

- **New** `apps/stats-web/src/lib/leaseCloseReason.ts` — `formatLeaseCloseReason()`, a pure helper mapping the chain's `LeaseClosedReason` to **viewer-neutral** labels (e.g. "Workloads unstable (provider)", "Provider decommissioned"), with numeric-range and raw-value fallbacks. Deliberately neutral, unlike deploy-web's tenant-centric "Closed by you".
- **New** `apps/stats-web/src/components/transactions/akash/MsgLeaseStartReclaim.tsx` — renders owner/dseq/gseq/oseq/provider + a friendly Reason row. Auto-wired into the dispatcher purely via the `akash/` barrel export (no dispatcher edit needed).
- **New** `leaseCloseReason.spec.ts` — 16 cases: each enum name, numeric/range fallbacks, omitted/zero/invalid sentinels, raw fallback.

No backend changes — the API already decodes the message generically (`msgToJSON`) and the indexer stores it generically.

**Verification (from `apps/stats-web`):** `npx tsc --noEmit` ✅ · `npm run lint -- --quiet` ✅ · `npm run test:unit` ✅ (37 passed)

stats-web has no component-test (RTL/jsdom) infrastructure — vitest runs `environment: "node"` and globs only `*.spec.ts` — so the component is verified by type-checking; its only logic is the unit-tested helper. Manual smoke once a real reclaim tx exists post-upgrade: open `/transactions/{hash}` and confirm the lease + Reason rows render instead of the JSON fallback.

## Release Notes

* **New Features**
  * Added UI component for displaying lease start reclaim transactions, including owner, provider, deployment details, and reclaim reason information
  * Implemented human-readable formatting for lease close and reclaim reasons to improve stats explorer usability

* **Tests**
  * Added comprehensive test coverage for lease close reason formatting with various input scenarios and edge cases

### CODE REVIEW

File: apps/stats-web/src/components/transactions/akash/MsgLeaseStartReclaim.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import { formatLeaseCloseReason } from "@/lib/leaseCloseReason";
+import { UrlService } from "@/lib/urlUtils";
+
+export const MsgLeaseStartReclaim = createMsgView(["v1beta5"], ({ message }) => {
+  const id = message?.data?.id;
+
+  return (
+    <>
+      <LabelValue label="Owner" value={<AddressLink address={id?.owner} />} />
+      <LabelValue label="dseq" value={<Link href={UrlService.deployment(id?.owner, id?.dseq)}>{id?.dseq}</Link>} />
```
[the remark this answers is not in the document]
Comment (baktun14) [reply apps/stats-web/src/components/transactions/akash/MsgLeaseStartReclaim.tsx:17]: Keeping this as-is for consistency. Every tx message component in `apps/stats-web/src/components/transactions/akash/` (`MsgCloseLease`, `MsgCreateDeployment`, `MsgDepositDeployment`, `MsgStartGroup`, `MsgPauseGroup`, `MsgUpdateDeployment`, `MsgCloseBid`, `MsgCreateBid`, `MsgCreateLease`, `MsgWithdrawLease`, `MsgCloseDeployment`, `MsgCloseGroup`, …) renders `<Link href={UrlService.deployment(...)}>` / `UrlService.address(...)` unconditionally with optional-chained args — none guard.

`MsgLeaseStartReclaim.id` is a required `LeaseID` on the decoded message, so for any real reclaim tx `owner`/`dseq`/`provider` are present; the `?.` here is only defensive against a wholly-missing `data`. Guarding solely this component would diverge from all siblings for no practical gain. If broken-href hardening is desired, it should be a separate, codebase-wide change (or centralized in `UrlService`), not a one-off here.

**Review by stalniy — APPROVED** [review]
